datavec/datavec-python/src/main/java/org/datavec/python/PythonJob.java

```diff
@@ -79,7 +90,7 @@ public void exec(PythonVariables inputs, PythonVariables outputs) throws Excepti
                 PythonObject arg = argsList.get(i);
                 PythonObject val = Python.globals().get(arg);
                 if (val.isNone()) {
-                    throw new RuntimeException("Input value not received for run() argument: " + arg.toString());
+                    throw new Exception("Input value not received for run() argument: " + arg.toString());
```

Comment:
Same thing as above. This was the wrong change. Create an explicit new class for this.